airsim_obstacle_msg[] obstacle_list
sensor_msgs/Image raw_image
